htdocs/expedition/class/expedition.class.php

```diff
@@ -1105,6 +1105,8 @@ function fetch_lines()
 				$obj = $this->db->fetch_object($resql);
 
                 $line->line_id          = $obj->line_id;
+                $line->rowid            = $obj->line_id;    // TODO deprecated
```

Comment:
If it's deprecated, shouldn't we rather update the rest of the code to use ```$line->id or $line->line_id```?
This would also reintroduce the deprecated field in 3.7 and 3.8 when merging.
Looks like a lot of duplication to me.